package.json

```diff
@@ -18,6 +18,7 @@
   "keywords": [
     "polygon.io",
     "stock api",
+		"options api",
```

Comment:
I think we're mixing tabs and spaces here. I'd check other files in this PR as well.